cross/openzwave/digests

```diff
@@ -0,0 +1,3 @@
+openzwave-r1009.tar.gz SHA1 6acc7bd5878b0e68df9bfa139c004c035b2ccf73
```

Comment:
With an svn checkout, this probably won't work for anything else but the file you originally created the digests with ;)